Readme.md

```diff
@@ -366,6 +316,12 @@ class Organization
      */
     public $projects = [];
 
+    public function __construct(string $identifier, string $name)
+    {
+        $this->identifier = $identifier;
+        $this->name = $name;
+    }
```

Comment:
This follows directly after the code example:
"Read Models currently need to be annotated with `@Flow\Entity`. At a later point it is planned to introduce a specific annotation `@CQRS\ReadModel` for that, but that requires a core change in Flow."

Any concrete idea how to improve?